chromiumcontent/chromiumcontent.gypi

```diff
@@ -122,7 +122,7 @@
           },
         },
       }],
-      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server"]', {
+      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server", "desktop_capture", "webrtc_common", "system_wrapper", "rtc_base", "rtc_base_approved", "libyuv"]', {
```

Comment:
It was done by [chromiumcontent.gypi#L93-L95](https://github.com/atom/libchromiumcontent/blob/master/chromiumcontent/chromiumcontent.gypi#L93-L95), this rule is to force gtk2ui to build as static library.